src/App.js

```diff
@@ -170,84 +171,140 @@ class App extends Component {
       <div className="App">
         <PageHeader>Invoice Generator</PageHeader>
         <p>
-          This is an invoice generator. Fill in the fields below and click 'Create Invoice' to generate the invoice as a PDF document. <a onClick={this.onExampleLinkClick}>Click here</a> to see an example.
+          This is an invoice generator. Fill in the fields below and click 'Create Invoice' to generate the invoice as a PDF document.
+          {' '}
+          <a onClick={this.onExampleLinkClick}>Click here</a>
+          {' '}
```

Comment:
Prettier did this when it shortened the line to make sure that the link is padded with spaces.